saleor/core/utils/random_data.py

```diff
@@ -632,6 +632,7 @@ def create_order_lines(order, discounts, how_many=10):
         )
         warehouse = next(warehouse_iter)
         increase_stock(line, warehouse, line.quantity, allocate=True)
+    manager.update_taxes_for_order_lines(order, lines)
```

Comment:
Are you sure of this? If someone will run populateDB after `cleardb` this will generate a huge amount of requests to external API. I would consider having dummy tax rates here instead of risking that we will call external API

Reply:
This function is implemented only in VatLayer. It does not generate any external API calls during the tax calculation process.